web/src/helpers.tsx

```diff
@@ -61,11 +61,6 @@ export const getMenuData = (): MenuDataItem[] => {
       path: '/ssl/list',
       icon: <IconFont name="iconssl" />,
     },
-    {
```

Comment:
Where?

Reply:
why remove setting?

Reply:
![image](https://user-images.githubusercontent.com/2106987/105807889-c312c680-5fe1-11eb-801b-96533c8d7e82.png)

The sidebar has too many tools, we could put Settings in Navbar.

Reply:
Got it.